frontend/site-registry/src/app/features/map/SiteMarkers/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC, useCallback, useMemo } from 'react';
+import { MapSearchQuery } from '../../../../graphql/generated';
+import MarkerClusterGroup from 'react-leaflet-cluster';
+import { SiteMarker } from './SiteMarker';
+import { StringParam, useQueryParam } from 'use-query-params';
+import { useMap } from 'react-leaflet';
+
+type Site = MapSearchQuery['searchSites']['sites'][number];
+interface SiteMarkersProps {
+  sites: Site[];
+}
+
+export const SiteMarkers: FC<SiteMarkersProps> = ({ sites }) => {
+  const map = useMap();
+
+  const [selectedSiteId, setSelectedSiteId] = useQueryParam(
+    'site',
+    StringParam,
+  );
+
+  const moveToSiteLocation = useCallback(
+    (site: Site) => {
+      map.flyTo(
+        {
+          lat: site.latdeg || 0,
+          lng: site.longdeg ? site.longdeg * -1 : 0,
```

Comment:
In the database, longitude values are positive, which is incorrect for locations in BC. I'll clarify if the prod data is the same way, and if so, we'll handle the conversion on the resolver level.

Reply:
I can cross check in the oracle TEST db and update, but if I can recollect there were methods for conversion written in Oracle

Reply:
Ideally I think we should be handling this during our data import but I understand that things are going to get very complicated especially since it looks like we're going to need to do a two way sync. If there's no simple way to handle this at the database level then this is probably the best option since we know that latitude is always positive and longitude is always negative.

In any case I think a comment is warranted because I think that anyone looking at this code would probably be confused with this logic.

Reply:
actually now that I'm thinking about it, it might be best to handle this on the backend when pulling the entities from the database if we can't do it on the database itself.

Reply:
I did find few conversion logics written in oracle, and also a conversion script written by a previous resource that works on the migration scripts https://github.com/bcgov/nr-site-registry/blob/dev/ora2pg/modify_geo.py. 
@anton-bcgov we can go over it to see if does what you need.

Reply:
@anton-bcgov  -  as discussed, we can keep the data in postgres db in the same format as what we receive now from oracle. This will also enable us to sync the data back to oracle seamlessly.